<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>KW:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="KW_l">
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_generieren">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Inhalt generieren</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/hammer_wrench_icon_135529.ico</normaloff>../../_Symbole/UsedSymbols/hammer_wrench_icon_135529.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="ss">
          <attribute name="title">
           <string>Gruppe 1</string>
          </attribute>
          <widget class="QLineEdit" name="lineEdit_1A">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 1A</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 1B</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_1B">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Gruppe 2</string>
          </attribute>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 2A</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2B">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_2A">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 2B</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Gruppe 3</string>
          </attribute>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 3A</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_3B">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_3A">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 3B</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Gruppe 4</string>
          </attribute>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 4A</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_4B">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_4A">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Gruppe 4B</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>English</string>
          </attribute>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>71</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>1 Lehrjahr</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>64</y>
             <width>71</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>2 Lehrjahr</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>71</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>3 Lehrjahr</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>71</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>4 Lehrjahr</string>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>20</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_2">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="Line" name="line">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>751</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_3">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>62</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_4">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>62</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_5">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>100</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_6">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>100</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_7">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>140</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="QTimeEdit" name="timeEdit_8">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>140</y>
             <width>61</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
          <widget class="Line" name="line_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>90</y>
             <width>751</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="Line" name="line_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>130</y>
             <width>751</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="Line" name="line_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>751</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="Line" name="line_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>170</y>
             <width>751</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_8">
          <attribute name="title">
           <string>BS-Abwesend</string>
          </attribute>
          <widget class="QLineEdit" name="lineEdit_BS">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Abwesenheit</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_9">
          <attribute name="title">
           <string>Homeoffice</string>
          </attribute>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Abwesenheit</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_Homeoffice">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>700</width>
             <height>80</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>80</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_abbrechen">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Abbrechen</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/closewindowapplication_cerca_ventan_2874.ico</normaloff>../../_Symbole/UsedSymbols/closewindowapplication_cerca_ventan_2874.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Speichern">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Speichern</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../_Symbole/UsedSymbols/kat.ico</normaloff>../../_Symbole/UsedSymbols/kat.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
